src/tests/ftest/daos_test/suite.yaml

```diff
@@ -55,6 +55,7 @@ server_config:
         - D_LOG_FLUSH=DEBUG
         - FI_LOG_LEVEL=warn
         - D_LOG_STDERR_IN_LOG=1
+        - ABT_THREAD_STACKSIZE=32768
```

Comment:
Should we instead put this here so all UCX tests use the same?
https://github.com/daos-stack/daos/blob/2469e2127cbfa8031d6241c0852082588a1c1817/src/tests/ftest/util/server_utils_params.py#L434-L447

Reply:
If we decide to land this, then I agree. Right now, it's just intended to determine if the change affects the outcome of the failing test. We anticipate an update to MOFED at some point that will include a UCX version that will not require increased stack allocation, so this PR may not be needed. I am converting the PR to draft status until the its utility is established.